src/ctap2/commands/mod.rs

```diff
@@ -314,7 +324,14 @@ impl Into<u8> for StatusCode {
 pub enum CommandError {
     InputTooSmall,
     MissingRequiredField(&'static str),
-    Deserializing(SerdeError),
+    Deserializing(CborError),
+    Serialization(CborError),
```

Comment:
Can we name these two consistently? Either Deerializing/Serializing, or Deserialization/Serialization